bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/NumericalSolver.cpp

```diff
@@ -108,6 +108,7 @@ inline void OpenSpaceToolkitAstrodynamicsPy_NumericalSolver(pybind11::module& aM
 
             .value("RungeKuttaCashKarp54", NumericalSolver::StepperType::RungeKuttaCashKarp54)
             .value("RungeKuttaFehlberg78", NumericalSolver::StepperType::RungeKuttaFehlberg78)
+            .value("RungeKutta4", NumericalSolver::StepperType::RungeKutta4)
```

Comment:
Because RK4 is more "basic", let's list it first:

```suggestion
            .value("RungeKutta4", NumericalSolver::StepperType::RungeKutta4)
            .value("RungeKuttaCashKarp54", NumericalSolver::StepperType::RungeKuttaCashKarp54)
            .value("RungeKuttaFehlberg78", NumericalSolver::StepperType::RungeKuttaFehlberg78)
```